src/css/main.less

```diff
@@ -209,13 +228,57 @@ a,
   padding: 0 0 5px 5px;
 }
 
-/* Application ID cell */
-.table > tbody > tr > td.appid {
-  display: flex;
+/* App list columns */
+.table > colgroup > col {
+
+  &.appid {
+    width: auto;
+  }
+
+  &.ram {
+    width: 96px;
+  }
+
+  &.cpu {
+    width: 80px;
+  }
 
-  .labels {
-    .label {
-      margin-left: @horizontal-spacing-unit;
+  &.status {
+    width: 216px;
+  }
+
+  &.tasks {
+    width: 48px;
+  }
+
+  &.health {
+    width: 6%;
+  }
+
+  &.actions {
+    width: 64px;
+  }
```

Comment:
I'm too sure about these guys. A few concerns:

1. mixing `%` and `px` values seems a bit arbitrary
2. adding these very specific classes (`.ram`, `.cpu` etc) inside the generic `.table` feels a bit "hacky". Should we maybe add a `.table.apps` or similar? 
3. we should definitely de-hardcode these values and use variables, or even better by simply multiplying our base-horizontal-spacing unit 

What do you think?